src/current/molt/molt-overview.md

```diff
@@ -0,0 +1,58 @@
+| [Verify](#verify)                                 | PostgreSQL, MySQL, CockroachDB        | Data validation                           | [Preview]({% link {{site.current_cloud_version}}/cockroachdb-feature-availability.md %}) |
+| [Live Migration Service](#live-migration-service) | PostgreSQL, MySQL, CockroachDB        | Consistent cutover                        | [Preview]({% link {{site.current_cloud_version}}/cockroachdb-feature-availability.md %}) |
+
+### Schema Conversion Tool
+
+The [MOLT Schema Conversion Tool]({% link cockroachcloud/migrations-page.md %}) [converts a source database schema]({% link {{site.current_cloud_version}}/migration-overview.md %}#convert-the-schema) to a CockroachDB-compatible schema. The supported Schema Conversion Tool sources are PostgreSQL, MySQL, Oracle, and SQL Server.
+
+The tool will convert [data definition (DDL) syntax]({% link {{site.current_cloud_version}}/sql-statements.md %}#data-definition-statements) (while ignoring destructive statements such as `DROP`), identify [unimplemented features and syntax incompatibilities]({% link {{site.current_cloud_version}}/migration-overview.md %}#unimplemented-features-and-syntax-incompatibilities) in the schema, and suggest edits according to CockroachDB [best practices]({% link {{site.current_cloud_version}}/migration-overview.md %}#schema-design-best-practices).
+
+### Fetch
+
+[MOLT Fetch]({% link molt/molt-fetch.md %}) moves data from a source database into CockroachDB. Data can be moved via one-time bulk ingestion and continuously via streaming ingestion. The supported Fetch sources are PostgreSQL, MySQL, and CockroachDB.
+
+You can use MOLT Fetch to [load test data]({% link {{site.current_cloud_version}}/migration-overview.md %}#load-test-data) into CockroachDB, enabling you to [test your application queries]({% link {{site.current_cloud_version}}/migration-overview.md %}#validate-queries) on CockroachDB. When you're ready to [conduct the migration]({% link {{site.current_cloud_version}}/migration-overview.md %}#conduct-the-migration) in a production environment, use MOLT Fetch to move your source data to CockroachDB. You can also enable continuous replication of any changes on the source database to CockroachDB.
+
+### Verify
+
+[MOLT Verify]({% link molt/molt-verify.md %}) checks for discrepancies between the source and target schemas and data. This involves verifying that table structures, column definitions, and row values match between the source and target. The supported Verify sources are PostgreSQL, MySQL, and CockroachDB.
+
+Use MOLT Verify after loading data with [MOLT Fetch](#fetch) and when [validating queries on CockroachDB]({% link {{site.current_cloud_version}}/migration-overview.md %}#validate-queries) to confirm that the CockroachDB data matches the source.
```

Comment:
What were we hoping to convey with the "validating queries on CockroachDB" part?  From my perspective, the sentence reads better without that part.

Reply:
This was a misunderstanding on my part -- I removed it from the diagram but neglected to cut it here. Thanks!